guide/matrix-vs-discord.md

```diff
@@ -74,32 +76,39 @@ Note that Matrix does not (and cannot, due to its decentralized nature) paywall
+| Organizing chats | "Servers" can be organized into folders, but each "server" can only belong to 1 folder. Channels in a "server" can only be organized by the "server" owner and moderators with "manage channels" permission, in a many to one fashion, and cannot be moved to another "server" once created. | Rooms can be included within an unlimited amount of Spaces. Spaces may also include other Spaces (similar to Discord's channel categories). |
+| Group chat privacy | Denying "View Channel History" permission prevents users from reading messages prior to their most recent login. However the "server" owners cannot enjoy that privacy, as "server" owners cannot deny their own permissions due to how Discord's permission system works. | You may deny new members from reading messages prior to them being invited / joining. You may also allow or deny guest access (such as [Matrix Static](http://view.matrix.org/)) from reading messages. You may also enable encryption[^4]. |
+| Server-side deletion guarantees | Messages that are removed from Discord are garbage collected from discord.com databases within less than 2 days.[^9] | As Matrix follows an event chain model, there is no true deletion. The closest thing available is redaction, which instructs the users and servers to blank the content but not the metadata of the event. Redactions are best-effort[^13]. |
+| Publicity | Although Discord offers its own "server" discovery feature, the requirements are somewhat arbitrary, so third-party services are often used. Only "server" owners can apply to Discord's own "server" discovery directory. Third party directory services may differ. | Each homeserver has a room directory which anyone in that homeserver may publish to. |
+| Invite | Through generating invite links or using an OAuth-based integration. | Through directly inviting users, or through shareable [addresses](../features/#promotion). |
+| Permissions in group chats | Single-owner, up to 255 roles. How long did it take for you to learn role hierarchy?[^8] A "server" can only be shut down by its owner, and that affects everyone. Members cannot demote their own permissions. Roles of a member are reset when a member leaves hence do not survive rejoin. | Up to 2^54 power levels (I think it's -2^53 to 2^53-1, however I highly doubt you will *ever* reach that limit), with minimal permissions. A user acquires a permission if their power level is equal to or higher than the power level required for the specific permission. Rooms are not owned by any user or server, hence cannot usually forcibly be shut down without coordination. Members can demote their own permissions. Power levels of members survive leave and rejoin. |
+| Size limits of group chats | 10 in group DMs, several tiers in "servers" ranging from 1000 to 1 million, subject to manual transistion across those tiers.[^15] | No artificial limits, albeit current implementations do not perform well past a few tens of thousands of members. | 
```

Comment:
```suggestion
| Size limits of group chats | 10 in group DMs, several tiers in "servers" ranging from 1000 to 1 million, subject to manual transition across those tiers.[^15] | No artificial limits, albeit current implementations do not perform well past a few tens of thousands of members. | 
```

Also for Matrix, I think it depends on server count rather than user count?

Reply:
User count. Guild count is already limited to 100-200/user for human users.